src/Cryptol/Parser/Position.hs

```diff
@@ -22,7 +22,10 @@ import Control.DeepSeq
 import Cryptol.Utils.PP
 
 data Located a  = Located { srcRange :: !Range, thing :: !a }
-                  deriving (Eq, Show, Generic, NFData)
+                  deriving (Eq, Generic, NFData)
+
```

Comment:
Generally we've tried to keep all the information in the `Show` instances, as they are mostly used for debugging.
Do we really need to drop the location info?